CONTRIBUTING.md

```diff
@@ -11,6 +11,15 @@ expectations the Node Foundation requires of its contributors.
 * A **Contributor** is any individual creating or commenting on an issue or pull request.
 * A **Collaborator** is a subset of contributors who have been given write access to the repository.
```

Comment:
Can this be submitted in another PR? These lines are not a part of this PR, and I'd like to focus the edits in this PR to the updates presented around the Getting Started guide 😅